__tests__/db.test.js

```diff
@@ -166,6 +166,27 @@ describe("GET /api/articles", () => {
         expect(msg).toBe("error! ID not found");
       });
   });
+  test("GET 200: responds with articles filtered by topics", () => {
+    return request(app)
+      .get(`/api/articles?topic=cats`)
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        expect(articles.length).toBeGreaterThan(0);
+        articles.forEach((article) => {
+          expect(article.topic).toBe("cats");
+        });
+      });
+  });
```

Comment:
One test missing here:
We have a valid topic with no associated articles (paper)
In this case we would expect 200 responds with an empty array